scripts/send.js

```diff
@@ -48,7 +48,9 @@ var messages = [
     to: 'Johnny Utah <johnny.utah@fbi.gov>',
     subject: 'The ultimate price',
     text: 'If you want the ultimate, you\'ve got to be willing to pay the ultimate price. \nIt\'s not tragic to die doing what you love.',
-    html: '<!DOCTYPE html><html><head></head><body style="background:#eeefont-family:sans-serifpadding:2em 2em">' +
+    html: '<!DOCTYPE html><html><head>' +
+          '<link rel="Shortcut Icon" type="image/x-icon" href="http://favicons.fr/icons_/105b697f95b9749509401e5b6eea24c0_preview_favicon-16.png" />' +
```

Comment:
There is a need to use local resources instead of online resources

Reply:
Some emails get a type="image/x-icon" in their body. half of that PR is to deiplay those icons. I don't see any other way to demonstrate that feature in thoses examples.